content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,296 @@
+
+
+## Create a project game release layout
+
+Now that you've prepared your project, you can create a project game release layout. There are different ways you can go about this, depending on how you set up your engine and whether you want to create a non-monolithic or monolithic release build.
+
+When you set up O3DE, you may have built your engine as a *source engine* or a *pre-built SDK engine*. You can use either build type to create a project game release layout. For more information on build types, refer to [Build the engine](https://o3de.org/docs/welcome-guide/setup/setup-from-github/#build-the-engine).
+
+Additionally, you can create a project game release layout that is either *non-monolithic* or *monolithic*. A monolithic release layout only contains files that are necessary to run the project, such as the Game Launcher, bundled content, and a minimal set of dependent `.dll` files, if any. A non-monolithic release layout contains the same files needed to run the project, as well as the objects that you use for development such as tools, shader compilers, and all `.dll` files.  Non-monolithic releases are typically much larger than monolithic releases. On some platforms that don't allow loading shared objects, you must use a monolithic release.
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory in your project directory that will contain the project's release layout.
+
+- Bundles your game content into an `engine.pak` file by packaging all of the product assets in your `Cache/pc` directory.
+
+- Builds the Game Launcher and other supporting files.
+
+{{< todo >}}
+The Game Launcher may fail to load your assets if the `engine.pak` file is over 4 GB. This issue will be resolved when development on the Asset Bundler is stable.
+{{< /todo >}}
+
+### Using source engine
+ 
+With the source engine, you must use a project-centric configuration to create a project game release layout. In a project-centric configuration, the CMake source directory is in the project build directory.
+
+The source engine supports non-monolithic projects by default. As listed under [Prerequisites](#prerequisites), you should have already created a Visual Studio project for the engine and registered the engine.
+
+#### Non-monolithic
+
+1. Use CMake to invoke Visual Studio to build your non-monolithic project. Specify the `INSTALL` target and `release` configuration.
+
+    ```cmd
+    cd C:/o3de-projects/MyProject
+    cmake --build build/windows_vs2019 --target INSTALL --config release. 
+    ```
+
+The result is a project release install directory that's located in your project's directory at `install/bin/Windows/release/Default`. Your packaged assets are located in `Cache/pc` in your project release install directory.
```

Comment:
I think we should clarify this by stating this is the directory is the specified to the `CMAKE_INSTALL_PREFIX` when configuring the project. If that variable wasn't set it defaults to the `<project-root>/install/bin/Windows/release/Default` directory.